scout/build/panel.py

```diff
@@ -1,94 +1,61 @@
 # -*- coding: utf-8 -*-
 import logging
-from datetime import datetime as datetime
 
 from scout.constants import VALID_MODELS
 from scout.exceptions import IntegrityError
 
 LOG = logging.getLogger(__name__)
 
 
-def build_gene(gene_info, adapter):
-    """Build a panel_gene object
+def build_gene(gene_info: dict, adapter) -> dict:
+    """Build a panel_gene object."""
 
-    Args:
-        gene_info(dict)
-
-    Returns:
-        gene_obj(dict)
-
-        panel_gene = dict(
-            hgnc_id = int, # required
-            symbol = str,
-
-            disease_associated_transcripts = list, # list of strings that represent refseq transcripts
-            reduced_penetrance = bool,
-            mosaicism = bool,
-            database_entry_version = str,
-            comment = str, # panel level gene comment str
-
-            ar = bool,
```

Comment:
This system was so messy, I've modified it so `inheritance_models` and `custom_inheritance_models` are now a list of strings, same as the final fields saved in the panel gene